src/Components/Common/Modal/Modal.module.scss

```
@@ -0,0 +1,61 @@
+.container {
+  width: 343px;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
+  height: auto;
+  border-radius: var(--radii-2);
+  background-color: var(--white);
+  position: fixed;
+  z-index: 9;
+  border: var(--gray400) 1px solid;
+  .header {
+    width: 100%;
+    height: var(--size-12);
+    margin: var(--size-7) 0 var(--size-5) 0;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    h2 {
+      font-size: var(--size-5);
+      font-weight: 600;
+    }
+    button {
+      background-color: transparent;
+      border: none;
+      position: absolute;
+      top: var(--size-10);
+      right: var(--size-3);
+      font-size: 15px;
+      cursor: pointer;
+    }
+  }
+  .content {
+    width: 100%;
+    margin: 0 auto;
+    height: auto;
+  }
+  .footer {
+    width: 307px;
+    margin: 0 var(--size-4) var(--size-10) var(--size-4);
+    justify-content: center;
+    align-items: center;
+  }
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```

```suggestion
  inset: 0;
```

inset으로 한 줄로 처리할 수 있습니다👍